operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -0,0 +1,794 @@
+#!/usr/bin/env bash
```

Comment:
Since this sourced into 'startServer.sh' is should be calling the equivalent of the startServer.sh 'exitOrLoop' instead of 'exit 1' throughout.

Reply:
@jshum2479 ?

Reply:
changed to return 1 if error and let caller calls exitOrLoop